### DISCUSSION TITLE
haarcascades for object detection

### DISCUSSION THREAD

**Discussion by Junaid-Amjad** [emgucv/emgucv#442] [Q&A]
Hi, i want to know is there any haarcascades for object detection. Right now, there are 2 xml files 
1. haarcascade_eye
2. haarcascade_frontalface_default
in the opencv>data>haarcascades

Another thing, why the xml file of [opencv](https://github.com/opencv/opencv/tree/master/data/haarcascades) is not compatible with emgucv. I tried using their haarcascade but i face exception during execution which i shared [here](https://github.com/emgucv/emgucv/issues/440)

**Comment by emgucv** [comment]
DNN face detection is faster and more accurate than HaarCascade:
https://github.com/emgucv/emgucv/blob/master/Emgu.CV.Models/Dnn/FaceDetector.cs

I would recommend using the DNN module over the older HaarCascade for face detection.

It also seems the xml file issues you mention has been resolved.

**Reply by emgucv** [reply]
You can try it out by building the Xamarin Forms demo in our release pacakge and running the "Facial Landmark Detection" demo.

**Reply by Junaid-Amjad** [reply]
Hi, i tried out Face Landmark Detection (DNN Module) but my objective is to detect object(car, person, animal etc) in the frame. I tried other demo available in the xamarin forms demo and found YOLO (DNN Module). Is there other solution available or i go with YOLO (DNN Module)?
